firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -16934,6 +16937,7 @@
 				D4C35391283500A600F7DC7D /* PerformanceTests.swift in Sources */,
 				B10D0B692D38F00E00925997 /* ShareLongPressTests.swift in Sources */,
 				2CF449A51E7BFE2C00FD7595 /* NavigationTest.swift in Sources */,
+				78A08C2E2D8CCD5300257E70 /* TabTrayTests.swift in Sources */,
```

Comment:
I would not add a new test suite just for this test, we have many tests related to tabs in Tab Tray in TopTabTest, it is true that the name is not very intuitive so I would add this test to that test suite and maybe rename it.

Reply:
Let me put the test in TabTrayTests. I may rename the file in a separate PR to keep the changes small.